### PR TITLE
update _meetings/intro-web.md

### PR DESCRIPTION
**Pull request by yaoj94**

### CODE REVIEW

**Comment by goelp14** [comment]
reminder to ian

File: _meetings/intro-web.md [hunk 1]
```diff
@@ -14,10 +14,9 @@ how-to-run: [
 	an alert(1) inside script tags, go to 'My Snippets', and click 'My site'."
 	]
 list-of-topics: [
-	"How clients and servers communicate via HTTP",
-	"There are lots of vulnerabilities within the server and client and
-	tools can help bypass protections",
-	"Introduced Google Gruyere and OverTheWire's Natas wargames"
+	"How clients and servers communicate via HTTP.",
```
Comment (ianklatzco) [inline _meetings/intro-web.md]: explained how

File: _meetings/intro-web.md [hunk 2]
```diff
@@ -14,10 +14,9 @@ how-to-run: [
...
+	"Warned about the the ethical and legal concerns of knowing how to hack website.",
```
Comment (ianklatzco) [inline _meetings/intro-web.md]: websites*

**Comment by ianklatzco** [comment]
thanks!!!

**Pull request merged**